### ISSUE TITLE
How to update NodeBB version?

### ISSUE THREAD

**Issue by surajdalvi** [NodeBB/NodeBB#3262]
Hi
 Currently, I am working on NodeBB **v0.6.0**  version but I want to update my NodeBB to **v0.7.0** version.without data loss. I am using redis as database.so please help me how to update NodeBB version?

**Comment by barisusakli** [comment]
Please see https://docs.nodebb.org/en/latest/upgrading/index.html

You need to first upgrade to the latest 0.6.x branch and from there to 0.7.0.

**Comment by surajdalvi** [comment]
Thanks.I follow all the steps and it works fine.I upgrade my NodeBB from **v0.6.0** to **v0.7.0**. But when I run my new NodeBB instance I get some warning for some plugins. for example:

- warn: [plugins/nodebb-plugin-markdown] This plugin may not be compatible with your version of NodeBB
. This may cause unintended behaviour or crashing.
- warn: [plugins/nodebb-plugin-markdown] In the event of an unresponsive NodeBB caused by this plugin,
 run ./nodebb reset plugin="nodebb-plugin-markdown".

I tried  **./nodebb reset plugin="nodebb-plugin-markdown"**  but  it does not work agin I got same warning. after that I manually upgrade plugin by using admin panel. but still I got same warning.

also,
I got one error -
**error: [sounds] Could not initialise sounds: EPERM, symlink '\NodeBB\public\sou
nds\notification.mp3'**
 
How I solve this?
